src/Ledger/Transaction.lagda

```diff
@@ -28,7 +28,7 @@ open import MyDebugOptions
 open import Relation.Nullary.Decidable using (⌊_⌋)
 
 data Tag : Set where
-  Spend Mint Cert Rewrd : Tag
+  Spend Mint Cert Rewrd Vote Propose : Tag
```

Comment:
If this creates all the visibility issues in this PR (i.e. `hiding (Vote)` everywhere), I would suggest naming these something else (e.g. `spend/mint/` or `SpendTag/MintTag/...`) to avoid such issues, unless this is super-important for the PDF presentation.

Reply:
The issue isn't the constructor itself, but the `Vote` type. There's both a `Vote` `Tag` and `ScriptPurpose`, and I went for the name `Vote` for consistency with all the other constructor names. I think my preferred option would be to not publicly reexport the `Vote` type from `TransactionStructure`.

Reply:
Sounds good!